beetsplug/lyrics.py

```diff
@@ -459,12 +508,14 @@ def __init__(self):
             'auto': True,
             'google_API_key': None,
             'google_engine_ID': u'009217259823014548361:lndtuqkycfu',
+            'genius_API_key': None,
             'fallback': None,
             'force': False,
             'sources': self.SOURCES,
         })
         self.config['google_API_key'].redact = True
         self.config['google_engine_ID'].redact = True
+        self.config['genius_API_key'].redact = True
```

Comment:
Can we use an all-lower-case config name, `genius_api_key`? I know this doesn't match the other options just above, but it does comport better with the rest of beets.